### ISSUE TITLE
Question: Documentation about GreenField- and/vs. PDFBox-implementation

### ISSUE THREAD

**Issue by r-pf** [veraPDF/veraPDF-library#956]
## Dev Effort
1D

## Description
Based on e.g. [#955](https://github.com/veraPDF/veraPDF-library/issues/955),  [#938](https://github.com/veraPDF/veraPDF-library/issues/938) and [#939](https://github.com/veraPDF/veraPDF-library/issues/939)  the following question:

Is there are documentation (present in the web) which describes the reason for a new/second implementation named GreenField and the advantages/disadvantages of it vs. the PDFBox-implementation?

**Comment by bdoubrov** [comment]
I don't think we put this explicitly to the web site. But the reasons are fairly simple:

- PDFBox is a proven PDF parser, but unfortunately it didn't fit into the PREFORMA project, which required that all the code should be licensed under MPLv2+ and GPLv3+. This excluded Apache license, as used by PDFBox

- the initial version of veraPDF validation was developed based on PDFBox library mainly because it allowed to test the validation model at a much earlier stage. Later on a a completely new (Grenfield) PDF parser was developed

- the goal is always to have both versions of veraPDF to be fully compatible. So, any difference in validation results are to be fixed. 

- most of the optimization efforts are spent now on the Greenfield parser. So, we normally recommend this version in long run. But PDFBox version is supported as well.

**Comment by bdoubrov** [comment]
See also the discussion in #923

**Comment by bdoubrov** [comment]
The support for PDFBox version will stop after the next release 1.28. It is strongly recommended to switch to the Greenfield version with the continued long-term support

**Comment by bdoubrov** [comment]
The release 1.30 does no longer support PDFBox version. Closing this issue for now
